src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Globalization;
```

Comment:
This is needed for the end tag generation - `writer.Write(string.Format(CultureInfo.InvariantCulture, "</{0}>", tagHelperOutput.TagName));`.

I already added a `using` for `Microsoft.AspNet.Razor.Runtime.TagHelpers`.